WebKitBrowser/CMakeLists.txt

```diff
@@ -225,6 +225,19 @@ target_link_libraries(${PLUGIN_WEBKITBROWSER_IMPLEMENTATION}
         WPEWebKit::WPEWebKit
         OpenSSL::Crypto)
 
+if (USE_DEVICESETTINGS)
```

Comment:
If you would like to have a flag, use it in both sourcecode and makefiles, Now you have "conditional" linking here and unconditional code. That does not make sense

Reply:
Addressed the review comments.